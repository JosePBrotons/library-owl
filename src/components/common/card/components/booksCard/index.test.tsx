import React from 'react';
import renderer from 'react-test-renderer';
import BooksCard from '.';

const props = {
    author: 'Emmie Thiel',
    title: 'Ring of Bright Water',
    imgUrl: 'http://wolox-training.s3.amazonaws.com/uploads/6942334-M.jpg',
};

test('renders book detail component correctly', () => {
    const booksCardCmp = renderer.create(<BooksCard {...props} />).toJSON();
    expect(booksCardCmp).toMatchSnapshot();
});
