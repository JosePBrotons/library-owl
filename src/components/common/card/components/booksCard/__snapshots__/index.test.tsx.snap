// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders book detail component correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "rgba(255, 255, 255, 1)",
      "borderRadius": 5,
      "elevation": 3,
      "flex": 1,
      "marginVertical": 10,
      "padding": 15,
      "shadowColor": "rgba(69, 69, 69, 0.5)",
      "shadowOffset": Object {
        "height": 2,
        "width": 2,
      },
      "shadowOpacity": 0.5,
      "shadowRadius": 3,
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
      }
    }
  >
    <Image
      source={
        Object {
          "uri": "",
        }
      }
      style={
        Object {
          "height": 90,
          "marginRight": 20,
          "width": 60,
        }
      }
    />
    <View
      style={
        Object {
          "justifyContent": "flex-start",
          "width": "70%",
        }
      }
    >
      <Text
        style={
          Object {
            "color": "rgba(69, 69, 69, 1)",
            "fontFamily": "Nunito-Bold",
            "fontSize": 16,
            "lineHeight": 22,
          }
        }
      >
        Ring of Bright Water
      </Text>
      <Text
        style={
          Object {
            "color": "rgba(106, 106, 106, 1)",
            "fontFamily": "Nunito-Regular",
            "fontSize": 16,
            "lineHeight": 22,
          }
        }
      >
        Emmie Thiel
      </Text>
    </View>
  </View>
</View>
`;
